test/bin/mdx-test/expect/without-cmis/test-case.md

```diff
@@ -0,0 +1,7 @@
+```ocaml
+# #require "stdio"
```

Comment:
I haven't been able to write a test case that does not use `stdio` or `base`, but that would be better. I think that this is mostly caused by type aliases in external packages.

Reply:
this can also wait for private library support and it should be easier to replicate with a private lib